config.toml

```diff
@@ -9,7 +9,7 @@ opts = { chainID = "1337", emitter = "0x3c747684333605408F9A4907DA043ee4c1A72D9c
 
 [[chains]]
 name = "centrifuge"
-type = "substrate"
+type = "ethereum"
 id = 1
 endpoint = "ws://localhost:8546"
 from = "029b67ec8aba36421137e22d874a897f8aa2a47e2d479d772d96ca8c5744b5a95c"
```

Comment:
Please ensure the scripts don't depend on this file

Reply:
I think we need to update our defaults in the config parsing